microshift_release_notes/microshift-4-12-release-notes.adoc

```diff
@@ -57,6 +57,8 @@ And reload the daemon.
 
 * There is an issue with the way {product-title}'s CNI driver manages the firewall rules. If the firewall settings are modified after {product-title} is started then {product-title} must be restarted. The recommended workaround is to configure firewall rules before starting {product-title}.
 
+* There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To work around this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
```

Comment:
```suggestion
* In the new installations of {product-title}, the `NodePort` service traffic might be stopped. To work around this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
```
Just a suggestion, based on the my understanding, to keep the sentence minimal. Feel free to ignore if the suggestion doesn't technically sounds correct.